addons/knobs/src/components/types/Object.js

```diff
@@ -88,12 +77,12 @@ ObjectType.defaultProps = {
 ObjectType.propTypes = {
   knob: PropTypes.shape({
     name: PropTypes.string,
-    value: PropTypes.object,
+    value: PropTypes.string,
   }),
   onChange: PropTypes.func,
 };
 
-ObjectType.serialize = object => JSON.stringify(object);
-ObjectType.deserialize = value => (value ? JSON.parse(value) : {});
+ObjectType.deserialize = value => eval(`(${value})`); // eslint-disable-line no-eval
```

Comment:
We need some XSS protection here

Reply:
Are we afraid of users scripting themselves or a knob url link with something potentially malicious inside it?

I think we can get around this by not allowing the javascript to be serialized into the URL if it's a real concern.

Reply:
Yeah, mostly the latter

Reply:
🎖 for you @Hypnosphi

Reply:
Actually it shouldn't be deserialized from URL, either